style-guide/properties.md

```diff
@@ -0,0 +1,34 @@
+# Property Value Subsection
+
+The Property Value subsection describes the value of a property. 
+
+## Guidelines
+
+In DxEditor, use the Return Value section (**returnValue** tag) for the property value description; DxEditor doesn't have a special tag for this subsection.
+
+- Provide a clear description of the property value. Keep the description concise, but provide enough context.
+- Use wording that is programming language-neutral.
+- If the property type is an enum member, briefly specify what the enum defines. The wording should distinguish between single enum values and flag enums, as shown in the following table.
+- If the property type is an array, indicate what the array contains.
+- If there is a default value that the user should be aware of, describe it in a second sentence.
+- Include measurement units (for example, "in bytes" or "in pixels") if applicable.
+- Don't include a link to the property value type in your descriptions. The build tools include this link automatically.
+- Do not bury important information about the property value in Remarks. Include it in the description, but point to Remarks for a lengthier discussion if necessary.
+
+The following table shows the boilerplate wording for property value descriptions. The wording varies according to the property type.
+
+| Property type | Wording | Examples |
+|---------------|---------|----------|
+| Class, interface, or structure | A/An/The &lt;*description, without specifying the data type*&gt;. The default is *default value*.
+  > [!NOTE]
```

Comment:
Notes don't work inside tables. I think we could probably do something like this to work around that:

```suggestion
  **Note:**<br />If the abstraction is not clear from the context, you can use the wording "An object that identifies/specifies/contains...". 
```

Reply:
Yeah, I thought that might be a problem, but wanted to see what it looked like. Changing